lib/sys/interactive.flow

```diff
@@ -200,3 +200,21 @@ handleRealEvents(l : EventHandler, clip : native, clipalive : ref bool, stage :
 	}
 }
 
+// Fallbacks for natives.
+getMouseX(clip : native) -> double {
+	println("TODO: implement 'getMouseX' native - a native fallback stub is used");
```

Comment:
Drop the " - a native fallback stub is used" just to save size in the output. The first part is sufficient for search/replace when it appears.

Reply:
Done